lib/phase/permutation.js

```diff
@@ -37,6 +37,10 @@ module.exports = validatedDoc => (
                 job.description = doc.jobs[jobName].description;
             }
 
+            if (doc.jobs[jobName].requires !== undefined) {
```

Comment:
Probably want to add a comment saying `Check for new config`

Reply:
Added comments